cluster/examples/kubernetes/ceph/upgrade-from-v0.9-delete.yaml

```diff
@@ -0,0 +1,201 @@
+---
```

Comment:
If these resources are being deleted, we can likely remove the `rules` to keep this shorter. I believe only the name/namespace of the resources are needed when being deleted.